collie/cross_validation.py

```diff
@@ -290,10 +293,11 @@ def _stratified_split_parallel_worker(idxs_to_split: Iterable[Any],
                                         stratify=np.ones_like(idxs_to_split))
     except ValueError as ve:
         if 'the resulting train set will be empty' in str(ve):
-            if ignore_warning is False:
+            if force_split is False:
                 raise ValueError(
-                    'Unable to straify split on users. The ``interactions`` object contains users '
-                    'with only one interaction, either set ``ignore_warning = True` or run '
+                    'Unable to stratify split on users. The ``interactions`` object contains users '
+                    'with a single interaction, either set ``force_split = True`` to put all users '
+                    'with a single interaction in the training set or run '
```

Comment:
Slight wording suggestion: 

```suggestion
                    'Unable to stratify split on users - the ``interactions`` object contains users '
                    'with a single interaction. Either set ``force_split = True`` to put all users '
                    'with a single interaction in the training set or run '
```